includes/Core/Modules/Module.php

```diff
@@ -132,19 +141,22 @@ abstract class Module {
 	 * @param User_Options   $user_options   Optional. User Option API instance. Default is a new instance.
 	 * @param Authentication $authentication Optional. Authentication instance. Default is a new instance.
 	 * @param Assets         $assets  Optional. Assets API instance. Default is a new instance.
+	 * @param Transients     $transients Optional. Transients instance. Default is a new instance.
 	 */
 	public function __construct(
 		Context $context,
 		Options $options = null,
 		User_Options $user_options = null,
 		Authentication $authentication = null,
-		Assets $assets = null
+		Assets $assets = null,
+		Transients $transients = null
 	) {
 		$this->context        = $context;
 		$this->options        = $options ?: new Options( $this->context );
 		$this->user_options   = $user_options ?: new User_Options( $this->context );
 		$this->authentication = $authentication ?: new Authentication( $this->context, $this->options, $this->user_options );
 		$this->assets         = $assets ?: new Assets( $this->context );
+		$this->transients     = $transients ?: new Transients( $this->context );
```

Comment:
With this available now, we should update the use in `Site_Verification` rather than instantiating it inline. Slightly out of scope here, but a very quick and easy refactor.